devbox.json

```diff
@@ -0,0 +1,23 @@
+{
+    "$schema": "https://raw.githubusercontent.com/jetify-com/devbox/0.13.7/.schema/devbox.schema.json",
+    "packages": [
+        "go",

```

Comment:
Should we pin the version of go to same version from go.mod?

Reply:
Sounds better, I'll add.